Formula/byobu.rb

```diff
@@ -4,31 +4,35 @@ class Byobu < Formula
   url "https://launchpad.net/byobu/trunk/5.133/+download/byobu_5.133.orig.tar.gz"
   sha256 "4d8ea48f8c059e56f7174df89b04a08c32286bae5a21562c5c6f61be6dab7563"
   license "GPL-3.0-only"
-  revision 1
+  revision 2
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "45b2277b96cd2c4c4170fbbe4a8ac92665062e8d0c835bfbd24ee3e3d52f2616"
   end
 
   head do
-    url "https://github.com/dustinkirkland/byobu.git"
+    url "https://github.com/dustinkirkland/byobu.git", branch: "master"
 
     depends_on "autoconf" => :build
     depends_on "automake" => :build
   end
 
-  depends_on "coreutils"
+  depends_on "gettext"
```

Comment:
I did see `byobu` scripts may run the gettext command, which would require `gettext` dependency. `gettext` shows up as a runtime dependency in:
- Debian package - https://packages.debian.org/sid/byobu
- Fedora package - https://packages.fedoraproject.org/pkgs/byobu/byobu/fedora-rawhide.html#dependencies

I doubt `coreutils` is needed on Linux and other Linux distros seem to be fine with the default installation.